pom.xml

```diff
@@ -107,9 +107,9 @@
 
     <!-- IntelliJ Annotations -->
     <dependency>
-      <groupId>com.intellij</groupId>
+      <groupId>org.jetbrains</groupId>
       <artifactId>annotations</artifactId>
-      <version>12.0</version>
+      <version>16.0.2</version>
     </dependency>
```

Comment:
Alternatively we could use
```
<dependency>
		<groupId>com.google.code.findbugs</groupId>
		 <artifactId>jsr305</artifactId>
		 <version>3.0.2</version>
	</dependency>
```
However, technically I don't see a huge benefit in changing this...the findbugs one might look more `native` java like...